src/vscode-utils.ts

```diff
@@ -58,6 +58,23 @@ export function isDocumentEmpty(): boolean {
     return !vscode.window.activeTextEditor?.document.getText.toString();
 }
 
+export function isValidFileExtensionInput(input: string): boolean {
+    const regExSingleExtension = new RegExp(/^\*\.[a-zA-Z]*(;)*/gm); // *.csv || *.csv;
+    const regExMultipleExtensions = new RegExp(/^\*\.[a-zA-Z]*(;)(\s)?(\*\.[a-zA-Z]*(;)*)*/gm); // *.csv;*.txt || *.csv;*.txt; || *.csv; *.txt;
+
+    const matchSingleExtension = input?.match(regExSingleExtension);
+    const matchMultipleExtensions = input?.match(regExMultipleExtensions);
+
+    let matchArray = matchSingleExtension;
```

Comment:
I would usually avoid using the type of a variable in its name. What about just `matches`?

Reply:
yes 👍